swift/sample-swift-sdk/sample-swift-sdk/Dashboards.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Dashboards.swift
+//  sample-swift-sdk
+//
+
+import Foundation
+import UIKit
+
+struct Dashboards {
+    var all:[DashboardModel] = []
+    
+    // Create struct that inherits identifiable for view
+    struct DashboardModel: Identifiable {
+        var id: Int64
+        var title: String
+        
+        init(id: Int64, title: String) {
+            self.id = id
+            self.title = title
+        }
+    }
+    
+    init() {
+        let appDelegate = UIApplication.shared.delegate as! AppDelegate
+        let sdk = appDelegate.sdk!
+        
+        // Grab the title and the content_metadata_id from the results
+        for dashboard in sdk.ok(sdk.all_dashboards(fields: "title,content_metadata_id")) {
```

Comment:
Wouldn't you just want `id` instead of `content_metadata_Id` if you're going to retrieve a dashboard (potentially)?

Reply:
Yeah `id`, was returning sparse results -- so I just went with `content_metadata_id`. You are probably right though that we want `id` instead